operators/src/processing/raster_vector_join/aggregator.rs

```diff
@@ -175,6 +191,37 @@ where
     fn is_satisfied(&self) -> bool {
         self.number_of_pristine_values == 0 || self.number_of_non_null_values == 0
     }
+
+    fn add_feature_data(&mut self, data: FeatureData, weight: u64) -> Result<()> {
```

Comment:
i think this could be a default implementation in the trait.

Reply:
see above